pandas/tests/extension/base/base.py

```diff
@@ -2,8 +2,18 @@
 
 
 class BaseExtensionTests:
+    @staticmethod
```

Comment:
Can you get rid of these methods altogether and just import / use as function within tests?

Reply:
I don't think so, since some tests need to override these

Reply:
Hmm OK - so was this change required? Looks like using `tm.assert_*` in the tests now?